src/vivarium_cluster_tools/psimulate/utilities.py

```diff
@@ -78,7 +79,8 @@ def set_permissions(output_dir):
     output_dir.chmod(permissions)
 
 
-def setup_directories(model_specification_file, result_directory, restart, expand):
+def setup_directories(model_specification_file: str, result_directory: str,
+                      restart: bool, expand: bool) -> Tuple[Path, Dict[str, Path]]:
```

Comment:
NIT: Think you can just do (Path, Dict[str, Path]) for the return type instead of using `Tuple` explicitly.